core/shared/src/main/scala/scalaz/zio/IOQueue.scala

```diff
@@ -135,11 +135,16 @@ class IOQueue[A] private (capacity: Int, ref: IORef[State[A]]) {
 object IOQueue {
 
   /**
-   * Makes a new queue.
+   * Makes a new bounded queue.
```

Comment:
We should add a comment here like:

"When the capacity of the queue is reached, any additional calls to `offer` will wait until there is more room in the queue."

Reply:
Done :)